doc/source/ray-job-submission/overview.rst

```diff
@@ -0,0 +1,228 @@
+============================
+From laptop to production
+============================
+
+**Ray Job Submission**
+
+**Concepts**
+
+- **Package**: A collection of files and configurations that defines an application thus allows it to be executed in a different environment remotely (ideally self-contained).
+
+- **Job**: A Ray application that can successfully execute on a user laptop, also can be submitted to a remote entity with specs such that its execution and lifetime is beyond a single ray cluster.
+Open source Ray will have the functionality to run jobs, but not manage them outside of a cluster.
+
+- **Job Manager**: An entity that manages the lifecycle of a Job and potentially multiple ray clusters, such as scheduling, killing, polling status, getting logs, and persisting inputs / outputs.
+Should be highly available (HA) by default
+Can be any framework with these abilities, such as Airflow.
```

Comment:
```suggestion
- **Job Manager**: An entity outside of the Ray cluster that manages the lifecycle of a job and potentially multiple ray clusters, such as scheduling, killing, polling status, getting logs, and persisting inputs / outputs. This could be any existing framework such as Airflow.
```